src/migration/pull-sharding-table-migration.js

```diff
@@ -49,6 +49,7 @@ class PullBlockchainShardingTableMigration extends BaseMigration {
                         ...nodes.slice(sliceIndex).filter((node) => node.nodeId !== '0x'),
                     );
                     sliceIndex = 1;
+                    // TODO: Should we fix it here also
```

Comment:
Not sure what is meant by this

Reply:
Not valid anymore, this comment should be removed